Neos.ContentRepository/Classes/Service/FallbackGraphService.php

```diff
@@ -31,13 +30,7 @@ class FallbackGraphService
 {
     /**
```

Comment:
TODO: Needed in Neos UI (for "whoever is interested"); not for internal use in CR.

TODO: maybe deprecate?